provider/openstack/cinder.go

```diff
@@ -56,7 +56,12 @@ func (env *Environ) cinderProvider() (*cinderProvider, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	return &cinderProvider{storageAdapter, env.name, env.uuid}, nil
+	return &cinderProvider{
+		storageAdapter,
```

Comment:
Heh, thanks for calling me out. I was just making the minimal change - will update to use field names.